pkg/runner/cluster_k8s.go

```diff
@@ -540,13 +548,13 @@ func (c *ClusterK8sRunner) createPod(ctx context.Context, podName string, input
 	defer c.pool.Release(client)
 
 	mountPropagationMode := v1.MountPropagationHostToContainer
-	hostpathtype := v1.HostPathType("DirectoryOrCreate")
-	sharedVolumeName := "s3-shared"
+	//hostpathtype := v1.HostPathType("DirectoryOrCreate")
+	sharedVolumeName := "efs-shared"
 
-	mnt := v1.HostPathVolumeSource{
-		Path: fmt.Sprintf("/mnt/%s/%s/%d", input.RunID, g.ID, i),
-		Type: &hostpathtype,
-	}
+	//mnt := v1.HostPathVolumeSource{
+	//Path: fmt.Sprintf("/mnt/%s/%s/%d", input.RunID, g.ID, i),
```

Comment:
This is a big difference - previously we had isolation since we mounted `/mnt/%s/%s/%d` to individual pods, whereas now we mount the full EFS file system to each pod and ask individual testplan instances to write to `/outputs/%s-%s-%s-%d` - not ideal :/